WcaOnRails/app/models/wcif.rb

```diff
@@ -13,6 +13,23 @@ def json
     {
       "formatVersion" => "1.0",
       "id" => competition.id,
+      "events" => competition.competition_events.map(&method(:competition_event_to_json)),
```

Comment:
Oh, I wasn't that clever at all. Just wrong. 

I didn't know there is a syntax for this!

I'd write it as `competition.competition_events.map{|ce| Wcif.competition_event_to_json(ce))}`